pkg/sentry/fsimpl/fuse/fusefs.go

```diff
@@ -207,23 +210,168 @@ type Inode struct {
 
 	locks vfs.FileLocks
 
-	dentry kernfs.Dentry
+	*dentry
+
+	// conn is used for communication between the FUSE server
+	// daemon and the sentry fusefs.
+	conn *Connection
```

Comment:
Might be better to avoid unnecessary copy. Can we use `inode.dentry.fs.conn`? Or we can add a method `Conn()` returning that